pkg/urbit/include/noun/vortex.h

```diff
@@ -13,8 +13,8 @@
         u3_noun wen;                      //  current time, as text
         u3_noun sev_l;                    //  instance number
         u3_noun sen;                      //  instance string
-        u3_noun our;                      //  identity
-        u3_noun fak;                      //  c3y is fake
+        u3_noun our;                      //  identity     XX Remove
+        u3_noun fak;                      //  c3y is fake  XX Remove
```

Comment:
Since this is going into a breach, these members can now be removed from this struct. (Doing so in a hotfix would break the memory image.)